scripts/createTestData.script

```diff
@@ -0,0 +1,290 @@
+use("test")
+
+// if specified, clear out all existing collections (except the users collection)
+if (true) {
+   db.volunteers.deleteMany({})
+   db.events.deleteMany({})
+}
+
+// creates 12 volunteerId variables
+for (let i = 1; i <= 12; i++) {
+  this[`volunteerId${i}`] = ObjectId();
+}
+
+db.volunteers.insertMany([
+{
+    "name" : "Jasmine Reyes",
+    "user_name" : "JasmineR",
+    "_id" : "",
```

Comment:
_id should be set to `volunteerId1` same with all the ids, except with the appropriate sequential number. You will also need to take a look at the new new schema when it's done and generate more matching test data. It's' probably best to wait until the schema is completely finalized before you do that. I know I already merged this, but I am just warning you about what will need to be changed in future iterations of this script. Also, I am wondering why we are setting the ids at all. The database would generate them anyway if we didn't.... I'll have to ask the higher ups.